pretrain.py

```diff
@@ -74,11 +76,17 @@ def gen_func(images):
 model = network.create(input_shape, args.network)
 network.compile(model, args.optimizer)
 
-model.save('pretrain.h5')
+# serialize model to JSON
```

Comment:
Why did you change this?

Reply:
As the ModelCheckpoint callback saves the full model, something similar to what model.save does, I thought the latter would be redundant.

Reply:
That sounds good, but why do we need to save the model as JSON? We probably can just remove the saving at the beginning altogether, no?

Reply:
@marco-c. Indeed, that was a mistake and it is already been removed, please check it now.